### DISCUSSION TITLE
v108+ fix doesnt work

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#318] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
when i put in `chrome.management.setEnabled('[extension id goes here]',!1)` as a bookmarklet, it shows up as `javscript:chrome.management.setenabled%28%27[extension id goes here]%27%2C %211%29`

if i run it directly it doesnt work either

**Comment by Alex2Mayhem** [comment, marked as answer]
> when i put in `chrome.management.setEnabled('[extension id goes here]',!1)` as a bookmarklet, it shows up as `javscript:chrome.management.setenabled%28%27[extension id goes here]%27%2C %211%29`
> 
> if i run it directly it doesnt work either

it was patched in 107. You can't run javascript in a bookmarklet anymoe.

**Reply by 3kh0** [reply]
use eruda, they did not patch and will not patch bookmarklets, it is just that your school blocked them
